ibm/service/project/resource_ibm_project_config.go

```diff
@@ -38,14 +38,158 @@ func ResourceIbmProjectConfig() *schema.Resource {
 				Type:        schema.TypeList,
 				MaxItems:    1,
 				Optional:    true,
-				Description: "A schematics workspace associated to a project configuration.",
+				Description: "A schematics workspace associated to a project configuration, with scripts.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"workspace_crn": &schema.Schema{
 							Type:        schema.TypeString,
 							Optional:    true,
 							Description: "An existing schematics workspace CRN.",
 						},
+						"validate_pre_script": &schema.Schema{
```

Comment:
We defined all these new arguments inside "schemaitcs" are they used during post or get operaitons

Reply:
all arguments inside "schematics" are populated by the backend logic and are only used for GET operations